File: python/ejercicios_loo/ejercicio_8.py
""" Se ingresa por teclado un conjunto de valores numéricos 
enteros positivos, se pide informar, por cada uno, 
si el valor ingresado es par o impar. 
Para indicar el final se ingresará un valor cero o negativo.
"""
v = int(input("Ingrese un valor entero positivo: "))

while v > 0:
    if v % 2 == 0:
        print("Es par")
    else:
        print("Es impar")
    v = int(input("Ingrese un valor entero positivo: "))

print("Fin")

File: java/ejercicios_loo/EjerciciosLOO/src/ejercicios/Ejercicio_8.java
import java.util.Scanner;

public class Ejercicio_8 {

    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        System.out.print("Ingrese un valor entero positivo: ");
        int v = entrada.nextInt();
        while (v > 0) {
            if (v % 2 == 0) {
                System.out.println("Es par");
            } else {
                System.out.println("Es impar");
            }
            System.out.print("Ingrese un valor entero positivo: ");
            v = entrada.nextInt();
        }
        entrada.close();

    }

}
